docs/source/conf.py

```diff
@@ -942,6 +942,7 @@
     "guard_scalar",
     "has_hint",
     "has_symbolic_sizes_strides",
+    "guard_size_oblivious",
```

Comment:
We've had churn with similar concepts maybe we want to wait before making it public ?

Reply:
I mean, this line just says to ignore this function when generating documentation lol

Reply:
I don't want to give this an underscored name and then have to rename it later when it "goes public". Let's just use the final name from the getgol.